pkg/ansible/ansible.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ansible
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/ava-labs/avalanche-cli/pkg/utils"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+)
+
+func CreateAnsibleHostInventory(inventoryPath, elasticIP, certFilePath string) error {
```

Comment:
Terraform has a templating feature. This might be easier than managing the file with Go. You can see an example of how we do things in Ash:
- [hosts.tftpl](https://github.com/AshAvalanche/ansible-avalanche-getting-started/blob/main/terraform/multipass/hosts.tftpl) (template)
- [local_file resource definition](https://github.com/AshAvalanche/ansible-avalanche-getting-started/blob/eae729cdcaa20382e8ab58849e11f294aefacfba/terraform/multipass/main.tf#L57) in the Terraform file